pkg/universe.dagger.io/x/wangwei27494731@gmail.com/kustomize/kustomize.cue

```diff
@@ -0,0 +1,59 @@
+package kustomize
+
+import (
+	"universe.dagger.io/bash"
+	"universe.dagger.io/docker"
+	"dagger.io/dagger"
+	"dagger.io/dagger/core"
+)
+
+// Apply a Kubernetes Kustomize folder
+#Kustomize: {
+	// Kubernetes source
+	source: string
+
+	// Optional Kustomization file
+	kustomization: string
+
+	// Kubeconfig
+	kubeconfig: dagger.#Secret
+
+	_baseImage: #Image
+
+	_file: core.#Source & {
+		path: source
+	}
+
+	_copy: docker.#Copy & {
+		input:    _baseImage.output
+		contents: _file.output
+		dest:     "/source"
+	}
+
+	_writeYaml: output: core.#FS
+
+	_writeYaml: core.#WriteFile & {
+		input:    dagger.#Scratch
+		path:     "kustomization.yaml"
+		contents: kustomization
+	}
+
+	_writeYamlOutput: _writeYaml.output
```

Comment:
You could directly exports it from `bash.#Run` with `exports.file."./kustomization.yaml"`